core/src/main/scripts/importer/validateData.py

```diff
@@ -4606,9 +4621,11 @@ def main_validate(args):
     else:
         portal_instance = load_portal_info(server_url, logger)
 
-    if args.portal_properties:
-        portal_instance.load_genome_info(args.portal_properties)
-
+    # specify species and genomic information
+    portal_instance.species = args.species
```

Comment:
there is no check that verifies if the given species is the same as the species already loaded in the portal. The code should check this. We can support multiple genomes in the same portal, but multiple species is an error situation. It is also fine if this check is done only in the Java layer. But there should be a check. It can be an easy check like (pseudo-code):
```
ref_genome_last_study = get last study's reference genome
if ref_genome_last_study.species != current_species
    abort
```

Reply:
what about you have a brand new database or load a new study? I think we should check against portal.properties file if needed.

Reply:
@khzhu if the DB is empty, then the check can be skipped.

Reply:
Hi @pieterlukasse , I do not think the validator can retrieve old version of the study. So, the validation of species should be added to CancerStudyReader util script the one does importing. The new method checkSpecies will do the check.
```
    private static Boolean checkSpecies(String studyId, String genomeName) {
        if (genomeName == null || genomeName == "") {
            return true;
        }
        try {
            CancerStudy oldCancerStudy = DaoCancerStudy.getCancerStudyByStableId(studyId);
            ReferenceGenome referenceGenome = DaoReferenceGenome.getReferenceGenomeByGenomeName(
                                oldCancerStudy.getReferenceGenome());
            return referenceGenome.getGenomeName().equalsIgnoreCase(genomeName);
        } catch (DaoException | NullPointerException e) {
            return true;
        }
    }
```